recipe/meta.yaml

```diff
@@ -26,11 +26,13 @@ source:
     # replace with placeholder that allows interested users to fix the file path
     - patches/0002-don-t-bake-non-relocatable-CMAKE_INSTALL_FULL_LIBDIR.patch
 
-# ensure cuda_compiler_version_min gets picked up by conda-smithy
+# ensure cuda_compiler* gets picked up by conda-smithy
+# [cuda_compiler == "something"]
+# [cuda_compiler_version == "something"]
 # [cuda_compiler_version_min == "something"]
```

Comment:
Looks like we may need more of these selector tricks to get re-rendering to work consistently here

Reply:
I have a fix. Can you please allow changes to this PR? It seems you unchecked the box for that

Reply:
FWIW this seems to work based on the re-rendering results below

That said, no strong feelings between this and another fix

Unfortunately since this repo lives in an org, GitHub doesn't allow that checkbox to work there

Can add you to the repo though. Here's [the invite link]( https://github.com/jakirkham-feedstocks/arrow-cpp-feedstock/invitations )

Reply:
Are you sure? The bot was able to push into #1084 and #1083, but not this one.

Reply:
> Can add you to the repo though. Here's [the invite link](https://github.com/jakirkham-feedstocks/arrow-cpp-feedstock/invitations)

Thanks for the invite, that circumvented the whole issue.